src/test-utils/dom/top-navigation/index.ts

```diff
@@ -101,10 +101,10 @@ export class TopNavigationMenuDropdownWrapper extends ButtonDropdownWrapper {
   }
 
   findTitle(): ElementWrapper | null {
-    return this.findByClassName(buttonDropdownStyles.title);
+    return createWrapper().findComponent(`.${buttonDropdownStyles.title}`, ElementWrapper);
```

Comment:
What is the reason of this change?

Reply:
Without this change, the test

```
TopNavigation Utility part › Menu dropdown › has a title in the dropdown
```

fails to find the title element, due to the menu dropdown having `expandToViewport`.